src/AddIns/Uno.UI.MediaPlayer.WebAssembly/HtmlMediaPlayer.cs

```diff
@@ -171,18 +171,13 @@ public void SetVolume(float volume)
 	{
 		add
 		{
-			if (_activeElement == null)
-			{
-				return;
-			}
-			_activeElement.RegisterHtmlEventHandler("timeupdate", value);
+			_htmlVideo.RegisterHtmlEventHandler("timeupdate", value);
+			_htmlAudio.RegisterHtmlEventHandler("timeupdate", value);
```

Comment:
Why did you need to change this? Is it because the playback may change type?

Reply:
Yes. 
The Playlist sample created was a mp4, mp3 and other mp4.
As the type change, we need to add the event to both types.
But worked fine.  
And I added a handling for null in javascript (MediaElement.ts).